stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/MessageListView.kt

```diff
@@ -781,14 +785,14 @@ public class MessageListView : ConstraintLayout {
     public fun init(channel: Channel) {
         this.channel = channel
         initAdapter()
-
-        messageListViewStyle = requireStyle().copy(
-            replyEnabled = requireStyle().replyEnabled,
-            threadsEnabled = requireStyle().threadsEnabled && channel.config.isThreadEnabled,
-        )
```

Comment:
Could we rename or update this method?
It is not an `init()` method anymore, it updates the channel instance and, the first time it is called, it initializes the adapter if it is not initialized.
It is a public method and could be a bit confusing

Reply:
Let's deprecate this method and create another one 👍